docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -49,22 +49,22 @@ These permissions are granted on an individual user basis as part of the user au
 
 If you sign up with GitLab Cloud, Codacy requires the following [permissions/scopes](https://docs.gitlab.com/ee/integration/oauth_provider.html#authorized-applications):
 
--   `api` permissions to access the authenticated user's API
--   `read_user` permissions to read the authenticated user's personal information
--   `read_repository` permissions to read the repositories
--   `openid` to authenticate using [OpenID Connect](https://docs.gitlab.com/ee/integration/openid_connect_provider.html#shared-information)
+-   `api` - Codacy uses GitLab's API to read and update pull requests, create webhooks for code push events, list commits, repositories, groups, members and permissions.
+-   `read_user` - Codacy retrieves the user's email addresses to enforce which commits are eligible for analysis.
+-   `read_repository` - Codacy retrieves repository metadata, such as name, languages and collaborators.
+-   `openid` - Codacy uses this permission for authentication using [OpenID Connect](https://docs.gitlab.com/ee/integration/openid_connect_provider.html#shared-information)
 
 ## Bitbucket Cloud
 
 If you log in with Bitbucket, Codacy requires the following [permissions/scopes](https://developer.atlassian.com/cloud/bitbucket/bitbucket-cloud-rest-api-scopes/):
 
--   Read and modify your account information
--   Read and modify your repositories' issues
+-   `account:write` - Codacy retrieves the user's email addresses to enforce which commits are eligible for analysis. Furthermore, Codacy creates an SSH key on the repository to allow cloning and integrating with your repository (see box above).
+-   `repository:admin` - Codacy retrieves repository metadata, such as name, languages and collaborators, and commit information. Codacy also creates an SSH key on the repository to allow cloning and integrating with your repository (see box above).
+-   `pullrequest:write` - Codacy retrieves pull request information to display on its side. Codacy might also create comments on the pull request, according to the results of code analysis
+-   `issue:write` -
+-   `webhook` - Codacy creates webhooks for code pushes and pull request events (created, merged, etc.). These events might trigger code analysis. Codacy also creates repository webhooks to enable [synced organizations](../organizations/what-are-synced-organizations.md).
+-   `team` - Codacy uses your group/team membership information to enforce permissions. 
 -   Read your workspace's project settings and read repositories contained within your workspace's projects
```

Comment:
It seems there is no correspondent scope, in the link at the start of this section, to this permission.